frontend/app/views/spree/shared/_footer.html.erb

```diff
@@ -53,16 +53,20 @@
           <% spree_menu('footer').children.each do |parent| %>
             <div class="col-3">
               <div class="footer-spree-label">
-                <%= link_to parent.name, spree_localized_item_link(parent) %>
+                <% if parent.item_type == 'Container' %>
+                  <%= parent.name %>
+                <% else %>
+                  <%= link_to parent.name, spree_localized_item_link(parent) %>
+                <% end %>
               </div>
               <% parent.children.each do |child| %>
-                <% if child.code == 'category' && child.item_type == 'Container' %>
-                  <% if child.children.present? %>
-                    <% child.children.each do |category_link| %>
-                      <div class="pt-2"><%= link_to category_link.name, spree_localized_item_link(category_link) %></div>
-                    <% end %>
+                <div class="pt-2">
+                  <% if child.new_window %>
+                    <%= link_to child.name, spree_localized_item_link(child), target: '_blank', rel: 'noopener noreferrer' %>
+                  <% else %>
+                    <%= link_to child.name, spree_localized_item_link(child) %>
```

Comment:
@MatthewKennedy let's create a new helper method `spree_nav_link_tag` and move logic there please eg. https://github.com/spree/spree/blob/master/frontend/app/helpers/spree/products_filters_helper.rb#L128